test/CMakeLists.txt

```diff
@@ -126,8 +126,6 @@ set(TEST_CASE_LIST)
 
 # Test that are leaky and may fail
 set(FAIL_TEST_CASES
-    test_24_backup
-    test_25_backup_compatibility
     test_26_backup_restore
```

Comment:
Should we also remove `test_26_backup_restore`?

Reply:
No, `test_24` and `test_25` check basic configuration such as initialization and simple discovery (acting as a normal server), which should always work. `test_26` checks more complex scenarios involving matching an alive client with a client loaded from the backup. This feature does not always work in spite of the raise in discovery times, so I think it's better to leave it as Flaky and address this issue in the future.